### ISSUE TITLE
2025-02-26, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#95]
# Today's Arxiv Papers

(1) [DRAMA: Diverse Augmentation from Large Language Models to Smaller Dense Retrievers](http://arxiv.org/abs/2502.18460v1), 2025-02-25

(2) [LLM-Based Design Pattern Detection](http://arxiv.org/abs/2502.18458v1), 2025-02-25

(3) [FRIDA to the Rescue! Analyzing Synthetic Data Effectiveness in Object-Based Common Sense Reasoning for Disaster Response](http://arxiv.org/abs/2502.18452v1), 2025-02-25

(4) [SWE-RL: Advancing LLM Reasoning via Reinforcement Learning on Open Software Evolution](http://arxiv.org/abs/2502.18449v1), 2025-02-25

(5) [Disambiguate First Parse Later: Generating Interpretations for Ambiguity Resolution in Semantic Parsing](http://arxiv.org/abs/2502.18448v1), 2025-02-25

(6) [MAPoRL: Multi-Agent Post-Co-Training for Collaborative Large Language Models with Reinforcement Learning](http://arxiv.org/abs/2502.18439v1), 2025-02-25

(7) [Reversal Blessing: Thinking Backward May Outpace Thinking Forward in Multi-choice Questions](http://arxiv.org/abs/2502.18435v1), 2025-02-25

(8) [TextGames: Learning to Self-Play Text-Based Puzzle Games via Language Model Reasoning](http://arxiv.org/abs/2502.18431v1), 2025-02-25

(9) [GLEAN: Generalized Category Discovery with Diverse and Quality-Enhanced LLM Feedback](http://arxiv.org/abs/2502.18414v1), 2025-02-25

(10) [When Benchmarks Talk: Re-Evaluating Code LLMs with Interactive Feedback](http://arxiv.org/abs/2502.18413v1), 2025-02-25

(11) [OmniAlign-V: Towards Enhanced Alignment of MLLMs with Human Preference](http://arxiv.org/abs/2502.18411v1), 2025-02-25

(12) [AgentRM: Enhancing Agent Generalization with Reward Modeling](http://arxiv.org/abs/2502.18407v1), 2025-02-25

(13) [Monte Carlo Temperature: a robust sampling strategy for LLM's uncertainty quantification methods](http://arxiv.org/abs/2502.18389v1), 2025-02-25

(14) [How Far are LLMs from Real Search? A Comprehensive Study on Efficiency, Completeness, and Inherent Capabilities](http://arxiv.org/abs/2502.18387v1), 2025-02-25

(15) [MindMem: Multimodal for Predicting Advertisement Memorability Using LLMs and Deep Learning](http://arxiv.org/abs/2502.18371v1), 2025-02-25

(16) [Responsible AI Agents](http://arxiv.org/abs/2502.18359v1), 2025-02-25

(17) [Which Contributions Deserve Credit? Perceptions of Attribution in Human-AI Co-Creation](http://arxiv.org/abs/2502.18357v1), 2025-02-25

(18) [BRIDO: Bringing Democratic Order to Abstractive Summarization](http://arxiv.org/abs/2502.18342v1), 2025-02-25

(19) [Mapping of Subjective Accounts into Interpreted Clusters (MOSAIC): Topic Modelling and LLM applied to Stroboscopic Phenomenology](http://arxiv.org/abs/2502.18318v1), 2025-02-25

(20) [WiCkeD: A Simple Method to Make Multiple Choice Benchmarks More Challenging](http://arxiv.org/abs/2502.18316v1), 2025-02-25

(21) [RefuteBench 2.0 -- Agentic Benchmark for Dynamic Evaluation of LLM Responses to Refutation Instruction](http://arxiv.org/abs/2502.18308v1), 2025-02-25

(22) [LDGen: Enhancing Text-to-Image Synthesis via Large Language Model-Driven Language Representation](http://arxiv.org/abs/2502.18302v1), 2025-02-25

(23) [DeepCircuitX: A Comprehensive Repository-Level Dataset for RTL Code Understanding, Generation, and PPA Analysis](http://arxiv.org/abs/2502.18297v1), 2025-02-25

(24) [AMPO: Active Multi-Preference Optimization](http://arxiv.org/abs/2502.18293v1), 2025-02-25

(25) [Better Aligned with Survey Respondents or Training Data? Unveiling Political Leanings of LLMs on U.S. Supreme Court Cases](http://arxiv.org/abs/2502.18282v1), 2025-02-25

(26) [Citrus: Leveraging Expert Cognitive Pathways in a Medical Language Model for Advanced Medical Decision Support](http://arxiv.org/abs/2502.18274v1), 2025-02-25

(27) [Iterative Counterfactual Data Augmentation](http://arxiv.org/abs/2502.18249v1), 2025-02-25

(28) [Unveiling and Causalizing CoT: A Causal Pespective](http://arxiv.org/abs/2502.18239v1), 2025-02-25

(29) [Debt Collection Negotiations with Large Language Models: An Evaluation System and Optimizing Decision Making with Multi-Agent](http://arxiv.org/abs/2502.18228v1), 2025-02-25

(30) [From ChatGPT to DeepSeek: Can LLMs Simulate Humanity?](http://arxiv.org/abs/2502.18210v1), 2025-02-25

(31) [LAG: LLM agents for Leaderboard Auto Generation on Demanding](http://arxiv.org/abs/2502.18209v1), 2025-02-25

(32) [Grandes modelos de lenguaje: de la predicción de palabras a la comprensión?](http://arxiv.org/abs/2502.18205v1), 2025-02-25

(33) [Intersubjective Model of AI-mediated Communication: Augmenting Human-Human Text Chat through LLM-based Adaptive Agent Pair](http://arxiv.org/abs/2502.18201v1), 2025-02-25

(34) [ChatMotion: A Multimodal Multi-Agent for Human Motion Analysis](http://arxiv.org/abs/2502.18180v1), 2025-02-25

(35) [Problem Solved? Information Extraction Design Space for Layout-Rich Documents using LLMs](http://arxiv.org/abs/2502.18179v1), 2025-02-25

(36) [SECURA: Sigmoid-Enhanced CUR Decomposition with Uninterrupted Retention and Low-Rank Adaptation in Large Language Models](http://arxiv.org/abs/2502.18168v1), 2025-02-25

(37) [Can LLMs Explain Themselves Counterfactually?](http://arxiv.org/abs/2502.18156v1), 2025-02-25

(38) [NusaAksara: A Multimodal and Multilingual Benchmark for Preserving Indonesian Indigenous Scripts](http://arxiv.org/abs/2502.18148v1), 2025-02-25

(39) [Jacobian Sparse Autoencoders: Sparsify Computations, Not Just Activations](http://arxiv.org/abs/2502.18147v1), 2025-02-25

(40) [Carbon and Silicon, Coexist or Compete? A Survey on Human-AI Interactions in Agent-based Modeling and Simulation](http://arxiv.org/abs/2502.18145v1), 2025-02-25

(41) [LevelRAG: Enhancing Retrieval-Augmented Generation with Multi-hop Logic Planning over Rewriting Augmented Searchers](http://arxiv.org/abs/2502.18139v1), 2025-02-25

(42) [Large Language Model Driven Agents for Simulating Echo Chamber Formation](http://arxiv.org/abs/2502.18138v1), 2025-02-25

(43) [Inverse Materials Design by Large Language Model-Assisted Generative Framework](http://arxiv.org/abs/2502.18127v1), 2025-02-25

(44) [HyperG: Hypergraph-Enhanced LLMs for Structured Knowledge](http://arxiv.org/abs/2502.18125v1), 2025-02-25

(45) [Bayesian Optimization for Controlled Image Editing via LLMs](http://arxiv.org/abs/2502.18116v1), 2025-02-25

(46) [Detecting Offensive Memes with Social Biases in Singapore Context Using Multimodal Large Language Models](http://arxiv.org/abs/2502.18101v1), 2025-02-25

(47) [Towards Thinking-Optimal Scaling of Test-Time Compute for LLM Reasoning](http://arxiv.org/abs/2502.18080v1), 2025-02-25

(48) [MRBTP: Efficient Multi-Robot Behavior Tree Planning and Collaboration](http://arxiv.org/abs/2502.18072v1), 2025-02-25

(49) [AutoCas: Autoregressive Cascade Predictor in Social Networks via Large Language Models](http://arxiv.org/abs/2502.18040v1), 2025-02-25

(50) [Harnessing Multiple Large Language Models: A Survey on LLM Ensemble](http://arxiv.org/abs/2502.18036v1), 2025-02-25

(51) [Detecting Knowledge Boundary of Vision Large Language Models by Sampling-Based Inference](http://arxiv.org/abs/2502.18023v1), 2025-02-25

(52) [AfroXLMR-Comet: Multilingual Knowledge Distillation with Attention Matching for Low-Resource languages](http://arxiv.org/abs/2502.18020v1), 2025-02-25

(53) [Verdict: A Library for Scaling Judge-Time Compute](http://arxiv.org/abs/2502.18018v1), 2025-02-25

(54) [NotaGen: Advancing Musicality in Symbolic Music Generation with Large Language Model Training Paradigms](http://arxiv.org/abs/2502.18008v1), 2025-02-25

(55) [Unveiling the Key Factors for Distilling Chain-of-Thought Reasoning](http://arxiv.org/abs/2502.18001v1), 2025-02-25

(56) [Multimodal Interaction and Intention Communication for Industrial Robots](http://arxiv.org/abs/2502.17971v1), 2025-02-25

(57) [LLM Knows Geometry Better than Algebra: Numerical Understanding of LLM-Based Agents in A Trading Arena](http://arxiv.org/abs/2502.17967v1), 2025-02-25

(58) [On Synthetic Data Strategies for Domain-Specific Generative Retrieval](http://arxiv.org/abs/2502.17957v1), 2025-02-25

(59) [Towards Better Understanding of Program-of-Thought Reasoning in Cross-Lingual and Multilingual Environments](http://arxiv.org/abs/2502.17956v1), 2025-02-25

(60) [DeepSeek-R1 Outperforms Gemini 2.0 Pro, OpenAI o1, and o3-mini in Bilingual Complex Ophthalmology Reasoning](http://arxiv.org/abs/2502.17947v1), 2025-02-25

(61) [Assessing Large Language Models in Agentic Multilingual National Bias](http://arxiv.org/abs/2502.17945v1), 2025-02-25

(62) [CaseGen: A Benchmark for Multi-Stage Legal Case Documents Generation](http://arxiv.org/abs/2502.17943v1), 2025-02-25

(63) [Advantage-Guided Distillation for Preference Alignment in Small Language Models](http://arxiv.org/abs/2502.17927v1), 2025-02-25

(64) [LeanProgress: Guiding Search for Neural Theorem Proving via Proof Progress Prediction](http://arxiv.org/abs/2502.17925v1), 2025-02-25

(65) [FACT-AUDIT: An Adaptive Multi-Agent Framework for Dynamic Fact-Checking Evaluation of Large Language Models](http://arxiv.org/abs/2502.17924v1), 2025-02-25

(66) [Scaling LLM Pre-training with Vocabulary Curriculum](http://arxiv.org/abs/2502.17910v1), 2025-02-25

(67) [Towards Sustainable Web Agents: A Plea for Transparency and Dedicated Metrics for Energy Consumption](http://arxiv.org/abs/2502.17903v1), 2025-02-25

(68) [Knowledge-enhanced Multimodal ECG Representation Learning with Arbitrary-Lead Inputs](http://arxiv.org/abs/2502.17900v1), 2025-02-25

(69) [VeriPlan: Integrating Formal Verification and LLMs into End-User Planning](http://dx.doi.org/10.1145/3706598.3714113), 2025-02-25

(70) [Can Large Language Models Identify Implicit Suicidal Ideation? An Empirical Evaluation](http://arxiv.org/abs/2502.17899v1), 2025-02-25

(71) [RankCoT: Refining Knowledge for Retrieval-Augmented Generation through Ranking Chain-of-Thoughts](http://arxiv.org/abs/2502.17888v1), 2025-02-25

(72) [Science Across Languages: Assessing LLM Multilingual Translation of Scientific Papers](http://arxiv.org/abs/2502.17882v1), 2025-02-25

(73) [Towards Enhanced Immersion and Agency for LLM-based Interactive Drama](http://arxiv.org/abs/2502.17878v1), 2025-02-25

(74) [SYNTHEMPATHY: A Scalable Empathy Corpus Generated Using LLMs Without Any Crowdsourcing](http://arxiv.org/abs/2502.17857v1), 2025-02-25

(75) [LR${}^{2}$Bench: Evaluating Long-chain Reflective Reasoning Capabilities of Large Language Models via Constraint Satisfaction Problems](http://arxiv.org/abs/2502.17848v1), 2025-02-25

(76) [Quantifying interdisciplinary synergy in higher STEM education](http://arxiv.org/abs/2502.17841v1), 2025-02-25

(77) [A Combinatorial Identities Benchmark for Theorem Proving via Automated Theorem Generation](http://arxiv.org/abs/2502.17840v1), 2025-02-25

(78) [Say Less, Mean More: Leveraging Pragmatics in Retrieval-Augmented Generation](http://arxiv.org/abs/2502.17839v1), 2025-02-25

(79) [MM-PoisonRAG: Disrupting Multimodal RAG with Local and Global Poisoning Attacks](http://arxiv.org/abs/2502.17832v1), 2025-02-25

(80) [A General Framework to Enhance Fine-tuning-based LLM Unlearning](http://arxiv.org/abs/2502.17823v1), 2025-02-25

(81) [Predicting Through Generation: Why Generation Is Better for Prediction](http://arxiv.org/abs/2502.17817v1), 2025-02-25

(82) [An Overview of Large Language Models for Statisticians](http://arxiv.org/abs/2502.17814v1), 2025-02-25

(83) [Can Multimodal LLMs Perform Time Series Anomaly Detection?](http://arxiv.org/abs/2502.17812v1), 2025-02-25

(84) [URO-Bench: A Comprehensive Benchmark for End-to-End Spoken Dialogue Models](http://arxiv.org/abs/2502.17810v1), 2025-02-25

(85) [DocPuzzle: A Process-Aware Benchmark for Evaluating Realistic Long-Context Reasoning Capabilities](http://arxiv.org/abs/2502.17807v1), 2025-02-25

(86) [Your Language Model May Think Too Rigidly: Achieving Reasoning Consistency with Symmetry-Enhanced Training](http://arxiv.org/abs/2502.17800v1), 2025-02-25

(87) [AIR: Complex Instruction Generation via Automatic Iterative Refinement](http://arxiv.org/abs/2502.17787v1), 2025-02-25

(88) [Exploring the Potential of Large Language Models for Estimating the Reading Comprehension Question Difficulty](http://arxiv.org/abs/2502.17785v1), 2025-02-25

(89) [Tip of the Tongue Query Elicitation for Simulated Evaluation](http://arxiv.org/abs/2502.17776v1), 2025-02-25

(90) [FoREST: Frame of Reference Evaluation in Spatial Reasoning Tasks](http://arxiv.org/abs/2502.17775v1), 2025-02-25

(91) [Uncertainty Quantification for LLM-Based Survey Simulations](http://arxiv.org/abs/2502.17773v1), 2025-02-25

(92) [DeepSeek vs. ChatGPT: A Comparative Study for Scientific Computing and Scientific Machine Learning Tasks](http://arxiv.org/abs/2502.17764v1), 2025-02-25

(93) [Design and implementation of a distributed security threat detection system integrating federated learning and multimodal LLM](http://arxiv.org/abs/2502.17763v1), 2025-02-25

(94) [Detection of LLM-Paraphrased Code and Identification of the Responsible LLM Using Coding Style Features](http://arxiv.org/abs/2502.17749v1), 2025-02-25

(95) [LLM Inference Acceleration via Efficient Operation Fusion](http://arxiv.org/abs/2502.17728v1), 2025-02-24

(96) [Aligning Compound AI Systems via System-level DPO](http://arxiv.org/abs/2502.17721v1), 2025-02-24

(97) [Spontaneous Giving and Calculated Greed in Language Models](http://arxiv.org/abs/2502.17720v1), 2025-02-24

(98) [Bridging Information Gaps with Comprehensive Answers: Improving the Diversity and Informativeness of Follow-Up Questions](http://arxiv.org/abs/2502.17715v1), 2025-02-24

(99) [Mind the Gesture: Evaluating AI Sensitivity to Culturally Offensive Non-Verbal Gestures](http://arxiv.org/abs/2502.17710v1), 2025-02-24

(100) [From Perceptions to Decisions: Wildfire Evacuation Decision Prediction with Behavioral Theory-informed LLMs](http://arxiv.org/abs/2502.17701v1), 2025-02-24
